command/export_test.go

```diff
@@ -0,0 +1,20 @@
+package command
+
+var (
+	NigoriName                 = nigoriName
```

Comment:
Why are we doing this?  Are you wanting to test this outside of this package?  I recommend keeping all tests that deal with this package in the package.  If these are configuration options, then we should have them configurable.

Reply:
Resolving this after discussed through slack DM.

The idea of putting tests in *_test package let us test package as a user of it, and these private var/methods needed by tests are exported to tests only through `export_test.go`.
We could found tests in the same or a separate *_test package patterns in go source code and there are arguments between which one is better.
Currently it seems like a developer preference and there are no real technical reasons at the moment to switch to the same package pattern, so we agreed to leave as is for now, resolving.